index.bs

```diff
@@ -3388,7 +3388,7 @@ partial interface MLGraphBuilder {
 <div class="note">
 <details open>
   <summary>
-    The behavior of this operation can be generically emulated via other operations as shown below, when the weight layout is the default {{MLGruWeightLayout/"zrn"}} layout, and the activation functions of the update/reset gate and new gate are of the operator types *sigmoid* and *tanh* respectively.
+    The behavior of this operation can be generically emulated via other operations as shown below, when the weight layout is the default {{MLGruWeightLayout/"zrn"}} layout, and the activation functions of the update/reset gate and new gate are of the operator types {{MLGraphBuilder/sigmoid()}} and {{MLGraphBuilder/tanh()}} respectively.
```

Comment:
Apologize I didn't mention in the last comment. We may also want to remove "of the operator types" here.

Reply:
done!